tests/test_core.py

```diff
@@ -1795,19 +1785,6 @@ def test_bigarray(self):
   def test_mod_globalstruct(self):
     self.do_run_in_out_file_test('tests', 'core', 'test_mod_globalstruct')
 
-  @no_wasm_backend('long doubles are f128s in wasm backend')
-  def test_pystruct(self):
```

Comment:
I wonder if this test is still useful?   It looks like it was disabled not because EMULATED_FUNCTION_POINTERS was missing but because of fp128.

Reply:
I think probably not... it just has some interesting structs in it, taken from python. We test equally interesting things in other places, and it's more of a clang-level feature anyhow. The interesting part of the test was the relocatable + emulated function pointers part which is removed, so I removed the whole test.